### PR TITLE
feat(mango): updated intro tutorial to use mango isntead of MapReduce

### PR DESCRIPTION
**Pull request by janl**
simplify the first-run experience

### CODE REVIEW

File: src/intro/tour.rst [hunk 1]
```diff
@@ -250,160 +250,109 @@ select Options, then check the Include Docs option. Finally, press the Run
...
+selector includes the actual query parameters that define what we are looking
+for exactly.
+
+Indexes are stored as rows that are kept sorted by the fields you specify. This
+makes retrieving data from a range of keys efficient even when there are
+thousands or millions of rows.
+
+Before we can run an example query, we'll need some data to run it on. We'll
+create documents with information about movies. Let's create documents for
+three movies. (Allow CouchDB to generate the _id and _rev fields.) Use Fauxton
```
Comment (flimzy) [inline src/intro/tour.rst:270]:
```suggestion
three movies. (Allow CouchDB to generate the ``_id`` and ``_rev`` fields.) Use Fauxton
```

File: src/intro/tour.rst [hunk 2]
```diff
@@ -250,160 +250,109 @@ select Options, then check the Include Docs option. Finally, press the Run
...
+             "year"
+          ]
+       },
+       "name": "year-json-index",
+       "type": "json"
+    }
 
-Name the design document ``_design/myDesignDoc``, and set the Index name
-to ``prices``.
+This defines an index on the field `year` and allows us to send queries for
```
Comment (flimzy) [inline src/intro/tour.rst:323]:
```suggestion
This defines an index on the field ``year`` and allows us to send queries for
```

File: src/intro/tour.rst [hunk 3]
```diff
@@ -250,160 +250,109 @@ select Options, then check the Include Docs option. Finally, press the Run
...
+Then click on ”Run Query”.
 
-Click "Save Document and then Build Index" and you should see result rows,
-with the various items sorted by price. This map function could be even more
-useful if it grouped the items by type so that all the prices for bananas were
-next to each other in the result set. CouchDB's key sorting system allows any
-valid JSON object as a key. In this case, we'll emit an array of [item, price]
-so that CouchDB groups by item type and price.
+The result should be a single result, the movie “My Neighbour Toto” which
+has the year value of 1988. `$eq` here stands for “equal”.
```
Comment (flimzy) [inline src/intro/tour.rst:341]:
```suggestion
has the year value of 1988. ``$eq`` here stands for “equal”.
```

File: src/intro/tour.rst [hunk 4]
```diff
@@ -250,160 +250,109 @@ select Options, then check the Include Docs option. Finally, press the Run
...
-View rows are sorted by their keys -- in this example, first by item,
-then by price. This method of complex sorting is at the heart of creating
-useful indexes with CouchDB.
-
-MapReduce can be challenging, especially if you've spent years working with
-relational databases. The important things to keep in mind are that map
-functions give you an opportunity to sort your data using any key you choose,
-and that CouchDB's design is focused on providing fast,
-efficient access to data within a range of keys.
+The result are the two movies from 1988 and 1998. `$lt` here means “lower than”.
```
Comment (flimzy) [inline src/intro/tour.rst:355]:
```suggestion
The result are the two movies from 1988 and 1998. ``$lt`` here means “lower than”.
```

**Comment by janl** [comment]
> some possible markdown-ish typos?

Most def, thanks for fixing.

**Pull request merged**